frontend/src/layouts/footer/Footer.tsx

```diff
@@ -34,17 +34,18 @@ const Watermark = styled("div")(({ theme }) => ({
 const Footer: React.FC = () => {
   const isDesktop = useResponsive({ query: "up", key: "md" });
   const [open, setOpen] = useState(false);
+  const theme = useTheme();
 
   return (
     <>
       <Divider />
-      <Paper sx={{ bgcolor: (theme) => theme.palette.background.neutral }}>
-        <Container sx={{ position: "relative", py: { xs: 8, md: 10 } }}>
+      <Paper sx={{ bgcolor: "background.neutral" }}>
+        <Container sx={{ position: "relative", py: { xs: 6, md: 10 } }}>
           <Grid container spacing={3} justifyContent={{ md: "space-between" }}>
             <Grid item xs={12} md={3}>
               <Stack alignItems="flex-start" spacing={3}>
                 <Logo />
-                <Typography variant="body3" sx={{ color: grey[700] }}>
+                <Typography variant="body3" sx={{ color: "text.secondary" }}>
```

Comment:
The change was made as the contrast between background.neutral and text.secondary is insufficient. The change was a poor one however, as grey[700] does not work for dark mode. 

Regardless, we need something with a bit more contrast here.

Reply:
Lighthouse testing doesn't complain about the contrast ratio, so it shouldn't be a UX or SEO problem.

I can agree to change it though. My main concern was the import of colors from mui, which doesn't apply the theme palette. The correct syntax in this scenario would be just "grey.700" as a string

Reply:
solved in [3015ad2](https://github.com/rubberdok/indok-web/pull/679/commits/3015ad28412e9b3f7d0a9de27b534a35e44a87ea)

Reply:
Wait, you didn't get complaints in LH? That's why I changed it 😮

Reply:
> I can agree to change it though. My main concern was the import of colors from mui, which doesn't apply the theme palette. The correct syntax in this scenario would be just "grey.700" as a string

TIL, thanks!